content/en/entities/Admin_DomainBlock.md

```diff
@@ -48,6 +49,13 @@ aliases: [
 **Version history:**\
 4.0.0 - added
 
+### `digest` {#digest}
+
+**Description:** The sha256 hex digest of the domain that is not allowed to federated.\
+**Type:** String\
+**Version history:**\
+4.3.0 - added
```

Comment:
I'm assuming commits only in `main` that add new fields will be 4.3.0 onwards.

Reply:
reasonable assumption (pending anything needing to be backported, for some reason)